webapp/channels/src/components/rhs_card_header/rhs_card_header.tsx

```diff
@@ -55,85 +54,42 @@ class RhsCardHeader extends React.PureComponent<Props> {
-                    <FormattedMessage
-                        id='rhs_header.backToResultsTooltip'
-                        defaultMessage='Back to search results'
-                    />
-                </Tooltip>
+            title = (
+                <FormattedMessage
+                    id='rhs_header.backToResultsTooltip'
+                    defaultMessage='Back to search results'
+                />
             );
             break;
         case RHSStates.FLAG:
-            backToResultsTooltip = (
-                <Tooltip id='backToResultsTooltip'>
-                    <FormattedMessage
-                        id='rhs_header.backToFlaggedTooltip'
-                        defaultMessage='Back to saved messages'
-                    />
-                </Tooltip>
+            title = (
+                <FormattedMessage
+                    id='rhs_header.backToFlaggedTooltip'
+                    defaultMessage='Back to saved messages'
+                />
             );
             break;
         case RHSStates.PIN:
-            backToResultsTooltip = (
-                <Tooltip id='backToResultsTooltip'>
-                    <FormattedMessage
-                        id='rhs_header.backToPinnedTooltip'
-                        defaultMessage='Back to pinned messages'
-                    />
-                </Tooltip>
+            title = (
+                <FormattedMessage
+                    id='rhs_header.backToPinnedTooltip'
+                    defaultMessage='Back to pinned messages'
+                />
             );
             break;
         }
 
-        const closeSidebarTooltip = (
-            <Tooltip id='closeSidebarTooltip'>
-                <FormattedMessage
-                    id='rhs_header.closeSidebarTooltip'
-                    defaultMessage='Close'
-                />
-            </Tooltip>
-        );
-
-        const expandSidebarTooltip = (
-            <Tooltip id='expandSidebarTooltip'>
-                <FormattedMessage
-                    id='rhs_header.expandSidebarTooltip'
-                    defaultMessage='Expand the right sidebar'
-                />
-                <KeyboardShortcutSequence
-                    shortcut={KEYBOARD_SHORTCUTS.navExpandSidebar}
-                    hideDescription={true}
-                    isInsideTooltip={true}
-                />
-            </Tooltip>
-        );
-
-        const shrinkSidebarTooltip = (
-            <Tooltip id='shrinkSidebarTooltip'>
-                <FormattedMessage
-                    id='rhs_header.collapseSidebarTooltip'
-                    defaultMessage='Collapse the right sidebar'
-                />
-                <KeyboardShortcutSequence
-                    shortcut={KEYBOARD_SHORTCUTS.navExpandSidebar}
-                    hideDescription={true}
-                    isInsideTooltip={true}
-                />
-            </Tooltip>
-        );
-
-        if (backToResultsTooltip) {
```

Comment:
I think its better to still have these as seperate variables since it improves a little code readability.